lib/board.rb

```diff
@@ -28,4 +28,14 @@ def cells
             "D4" => Cell.new("D4")
     }
     end
-end
+
+    def valid_coordinate?(coordinate)
+        valid_row = ["A", "B", "C", "D"]
```

Comment:
Makes sense. My only thought is that we would need to refactor if there was a scenario in which the board changes. In that case, we could consider iterating over the cells hash to check if the coordinate is present on the board we created.